strings/accent_universal.json

```
@@ -22,6 +22,7 @@
 		"ohio": "underworld",
 		"fanum": "demon",
 		"cenat": "demon",
+		"jerma": "demon",
```

```suggestion
		"jerma": "xylix",
```
the devilish trickster @PPooch